ddp-workspace/projects/ddp-dsm-ui/src/app/sharedLearningUpload/components/uploadFile/uploadFile.component.ts

```diff
@@ -77,7 +79,12 @@ export class UploadFileComponent implements OnDestroy {
         )
         .subscribe({
           next: () => this.handleSuccess(somaticResultsFile),
-          error: (error: any) => this.handleError(error)
+          error: (error: any) => {
+            // if there's an error storing the file, remove the file
+            // and inform the user to retry
```

Comment:
Is there an indication of an error, or does the button just say 'Retry'?

Reply:
Updated to include a new `ERROR_RETRY` enum to differentiate regular retry from retrying due to error.